packages/experimental/kai/src/hooks/once.ts

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+export const once = (storage: any) => (operationKey: string, fn: Function) => async () => {
```

Comment:
this doesn't look like a hook, so it should probably be moved elsewhere

Reply:
see above.